README.md

```diff
@@ -11,29 +11,73 @@ theory behind these constructors is researched in
-polishing rough edges, improving diagnostics, improving compilation
-speed, and embedding in other tools.
-
-We expect that, over time, some of the new technologies explored in
-this project will find their way into future versions of Scala. At
-present it is too early to say which ones and when.
-
-If you want to try it out, to get started have a look at
+####Current status:
+ _Technology preview_: currently unsupported, may not be functionally complete, and are not suitable for deployment in production.
+
+####Is it going to be the future Scala?
+Yes, eventually.
+
+####Who's working on it?
+See [github contributors page](https://github.com/lampepfl/dotty/graphs/contributors).
+ 
+####What are the features that could make me consider trying it?  
+| Feature                                         	| Status              	|
+|-------------------------------------------------	|---------------------	|
+| Union, Intersection and Literal singleton types 	| Implemented         	|
+| Fast compilation (phase fusion)                 	| Implemented         	|
+| Working contravariant implicits                 	| Implemented         	|
+| Trait parameters                                	| Implemented         	|
+| @Static methods and fields                      	| Implemented         	|
+| Colored Repl                                    	| Implemented         	|
+| Sbt incremental build                           	| Implemented         	|
+| Non-blocking lazy vals                          	| Implemented         	|
+|                                                 	|                     	|
+| Non-boxed arrays of value classes               	| In progress         	|
+| Auto-Specialization                             	| [In progress](https://github.com/dotty-linker/dotty)         	|
+| Whole program optimizer                         	| [In progress](https://github.com/dotty-linker/dotty)         	|
+| Library defined optimizations                   	| [In progress](https://github.com/dotty-linker/dotty)         	|
+|                                                 	|                     	|
+| HList & HMaps\Record types                      	| Under consideration 	|
+| Implicit functions                              	| Under consideration 	|
+| Effects                                         	| Under consideration 	|
+| Auto-completion in repl                         	| Under consideration 	|
+| Spec name-based patmat                          	| Under consideration 	|
+| Multiverse equality                             	| Under consideration 	|
+| Exhaustivity checks in pattern matching          | Under consideration 	|
+There are also plethora of small details such [per-callsite @tailrec annotations](https://github.com/lampepfl/dotty/issues/1221)
+
+####What are the complications that I can have If I start using Dotty?
+Dotty can use libraries compiled by scalac 2.11, but Scala scalac can't use libraries compiled by Dotty.<br>
+No existential types.<br>
```

Comment:
I'd say "No existential types using the `forSome` syntax, most of the times these can be rewritten using wildcards (`_`) instead."